frontend/components/groups/js/groups.service.ts

```diff
@@ -329,6 +329,16 @@ export class GroupsService {
 		return this.deleteGroups(teamspace, model, groupsToDelete);
 	}
 
+	/**
+	 * Deletes filtered groups in the current model
+	 * @param teamspace the teamspace name for the group
+	 * @param model the model id for the group
+	 */
+
+	public deleteFilteredGroups(teamspace: string, model: string, filteredGroups: any) {
```

Comment:
Instead of creating this function, we should rename `deleteAllGroups` to `deleteAllDisplayedGroups`. We always only want to delete the groups that are currently displayed. If there is no filter, `groups` would be teh same as `groupsToShow`

So a new function is unnecessary. (but it would be good to change all the prompts from `delete all` to `delete all displayed` or something

Reply:
I opted for the deleteGroups method , much less code and only being called once in groups.component.

Reply:
We need to delete this method as it's not used